source/_integrations/screenlogic.markdown

```diff
@@ -46,6 +46,27 @@ Sets the operation of any connected color-capable lights.
 | ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `target`               | no       | An `area` containing the ScreenLogic device, the ScreenLogic `device` itself, or any `entity` from the ScreenLogic device you wish to set the color mode on. |
 | `color_mode`           | no       | The color mode to set. Valid values are listed below.                                                                                                        |
+## Examples
+
+### Current Pool/Spa Temperature
+
+Some may wish to have the current pool or spa temperature as a separate `sensor` entity in addition to the existing `climate` entity. This can be done with a [template sensor](/integrations/template):
+
+{% raw %}
+
+```yaml
+# Example template sensor
+template:
+  - sensor:
+      - name: "Pool Temperature"
+        # Replace the climate entity with your climate entity
+        state: "{{ state_attr('climate.pentair_XX_XX_XX_pool_heat', 'current_temperature') }}"
```

Comment:
This doesn't seem ScreenLogic/pentair specific. It might be better on the template sensor page instead if there isn't something similar there already

Reply:
There's an example of exposing a state attribute as a separate sensor in the [Sun Angle example](https://www.home-assistant.io/integrations/template/#state-based-sensor-exposing-sun-angle), but nothing specifically about exposing the current temperature from a `climate` entity.

Would you prefer I add a generic example there?

Reply:
I agree with bdraco here, this is not specific to this integration.

Therefore, going to friendly decline this PR.